implementation/UserValidation.py

```
@@ -1,13 +1,14 @@
-               return self.cursor.collection('user_model_table').document(mail)
+        def userValidate(self,mail:str): 
+               result=requests.get(f"http://localhost:7006/fetch/{mail}").json()        
+               return result
         
-        def validateUser(self,dataModel:dict)->bool:
-                self.doc=self.userValidate(dataModel['mail'])
-                doc=self.doc.get()
-                return True if not doc.exists else False 
+        def validateUser(self,mail:str)->bool:
+                self.doc=self.userValidate(mail)
+                return True if self.doc['status']=='ok' else False 
```

Ruff (0.11.9)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)